index.bs

```diff
@@ -1508,7 +1516,38 @@ used to help facilitate isolating problems with a specific version of a device.
 If the attestation root certificate is not dedicated to a single WebAuthn Authenticator device line (i.e., AAGUID), the AAGUID
 should be specified in the attestation certificate itself, so that it can be verified against the <a>authenticatorData</a>.
 
+## Algorithm to Relax the Same-Origin Restriction ## {#algo-relax-same-origin-restriction}
```

Comment:
I suggest moving this, from here in (the present) section 5 {#authenticator-model}, to (the bottom of?) section 4 {#api}.  it is not properly part of the authnr model AIUI.